examples/shader/shadertree.py

```diff
@@ -12,27 +12,21 @@
 
 '''
 
-from kivy.clock import Clock
 from kivy.app import App
-from kivy.uix.button import Button
-from kivy.uix.scatter import Scatter
-from kivy.uix.floatlayout import FloatLayout
-from kivy.core.window import Window
+from kivy.clock import Clock
+from kivy.graphics import (RenderContext, Fbo, Color, ClearColor, ClearBuffers,
+        Rectangle)
 from kivy.properties import StringProperty, ObjectProperty
-from kivy.graphics import RenderContext, Fbo, Color, Rectangle
+from kivy.uix.floatlayout import FloatLayout
 
+# imported for side effects needed by Shader
+import kivy.core.window
```

Comment:
I'm not sure, but why `from kivy.core.window import Window` isn't good (i.e. the old version)?

Also, (as you aren't the only one with such PR) I don't really think using [this feature](https://www.jetbrains.com/help/pycharm/optimizing-imports.html) that alphabetically sorts all imports is actually useful as it introduces unnecessary noise to the diff. If you could, it'd be lovely to get a diff without this noise, please. Just a silly wish ^-^